tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py

```diff
@@ -269,21 +269,14 @@ def create_tf_checkpoint(tmp_dir):
 def create_tf_function(temp_dir):
```

Comment:
I think it makes sense to have functions and tests for both `keras.Model` and `ConcreteFunction`

Reply:
We have keras.Model test here: https://github.com/openvinotoolkit/openvino/blob/656d7fe380e8a5d8c245a47c2c0e41b56baddeb7/tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py#L40